coldfront/core/allocation/management/commands/add_allocation_defaults.py

```diff
@@ -13,7 +13,7 @@ class Command(BaseCommand):
     def handle(self, *args, **options):
 
         for attribute_type in ('Date', 'Float', 'Int', 'Text', 'Yes/No', 'No',
-            'Attribute Expanded Text'):
+            'Attribute Expanded Text', "JSON", "BillingCycle"):
```

Comment:
@peaches7083 [MINOR] 
Could you refactor the list to a variable or constant?

Reply:
Can do.

Reply:
Done.